### PR TITLE
Fixed Typos in README.rst

### PR DESCRIPTION
**Pull request by prjailani**
### Description

There are a few grammatical mistakes in the Readme file. So, I fixed those.

Fixes #

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x} Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by prjailani** [comment]
phpMyAdmin Developer's Certificate of Origin. Version 1.0

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the
    right to submit it under the license of "GNU General Public License or
    any later version" ("GPLv2-or-later"); or

(b) The contribution is based upon previous work that, to the best of my
    knowledge, is covered under an appropriate open source license and I have
    the right under that license to submit that work with modifications,
    whether created in whole or in part by me, under GPLv2-or-later; or

(c) The contribution was provided directly to me by some other person who
    certified (a) or (b) and I have not modified it.

(d) I understand and agree that this project and the contribution are public
    and that a record of the contribution (including all metadata and
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    phpMyAdmin's policies and the requirements of the GPLv2-or-later where
    they are relevant.

(e) I am granting this work to this project under the terms of the
    GPLv2-or-later.

    https://www.gnu.org/licenses/gpl-2.0.html

    Signed-off-by: Abdul Kader Jailani P R <jailanijailani968@gmail.com>

File: README.rst [hunk 1]
```diff
@@ -53,10 +53,10 @@ You can get the newest release at https://www.phpmyadmin.net/.
 If you prefer to follow the Git repository, the following branch and tag names may be of interest:
 
 * ``STABLE`` is the current stable release.
-* ``master`` is the development branch.
-* Releases are tagged, for example version 5.0.1 was tagged as ``RELEASE_5_0_1``.
+* ``Master`` is the development branch.
```
Comment (MauricioFauth) [inline README.rst:56]:
```suggestion
* ``master`` is the development branch.
```

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Comment by prjailani** [comment]
Thanks for pointing me out @MauricioFauth . I am just in the state of finding typos 😂.

**Review by williamdes — APPROVED** [review]

**Comment by prjailani** [comment]
@MauricioFauth and @ibennetch Why it's still not merged. Any problem in it? Thanks.

**Review by MauricioFauth — APPROVED** [review]

**Comment by MauricioFauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**